src/pages/community.md

```diff
@@ -29,11 +34,11 @@ Join <a href="https://slack.rancher.io"><img src="https://a.slack-edge.com/fd21d
 [#k3s-contributor](https://rancher-users.slack.com/archives/CGXR87T8B) - Contribution discussions
 
 ## Follow us!
-Bluesky:
+[Bluesky](https://bsky.app/profile/k3sio.bsky.social)
 
-X:
+[X](https://x.com/k3sio_io)
```

Comment:
ugh, do we have to use X? I'd prefer to have a Mastodon account for the project, if the Linux Foundation has a server they suggest.